### PR TITLE
galaxy importer lint and fixes

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: galaxy.yml [hunk 1]
```diff
@@ -79,6 +79,7 @@ tags:
   - katello
   - satellite
   - orcharhino
+  - infrastructure
```
Comment (evgeni) [inline galaxy.yml:82]: the full list of tags is:
- application
- cloud
- database
- infrastructure
- linux
- monitoring
- networking
- security
- storage
- tools
- windows

We're required to have *one* of these, and I think "infrastructure" fits best to Foreman and thus this collection.

**Review by mdellweg — APPROVED** [review]
Very nice!!!

**Comment by mdellweg** [comment]
Should you add `importer_result.json` to `.gitignore`?

**Comment by evgeni** [comment]
> Should you add `importer_result.json` to `.gitignore`?

Yes! https://github.com/theforeman/foreman-ansible-modules/pull/1323

**Pull request merged**